frontend/e2e-test/test/e2e/dev-api/index.ts

```diff
@@ -580,3 +580,91 @@ export async function cleanUpApplicationEmails(): Promise<void> {
     throw new DevApiError(e)
   }
 }
+
+type PairingStatus =
+  | 'WAITING_CHALLENGE'
+  | 'WAITING_RESPONSE'
+  | 'READY'
+  | 'PAIRED'
+
+export interface PairingResponse {
+  id: UUID
+  unitId: UUID
+  challengeKey: string
+  responseKey: string | null
+  expires: Date
+  status: PairingStatus
+  mobileDeviceId: UUID | null
+}
+
+export async function postPairingChallenge(
+  challengeKey: string
+): Promise<PairingResponse> {
+  try {
+    const { data } = await devClient.post<PairingResponse>(
```

Comment:
This is also fine, but I'm wondering if we could do these things through standard API without duplicating these to dev-api. Maybe @Gekkio can comment?